src/Lecture2.hs

```diff
@@ -156,6 +176,21 @@ data Knight = Knight
     , knightAttack    :: Int
     , knightEndurance :: Int
     }
+{-
+data DragonColor 
+    = Red
+    | Black
+    | Green
+
+data Dragon = Dragon
+    { dragonHealth    :: Int
+    , dragonAttack    :: Int
+    , dragonEndurance :: Int
+    }
```

Comment:
That's an excellent start so far 👍🏻 
Any immediate difficulties or just in progress? 🙂

Reply:
Just in progress :-)